projects/sample-widget/src/lib/main.ts

```diff
@@ -49,52 +118,212 @@ template.innerHTML = `
     }
 
     ::-webkit-scrollbar-thumb {
-        border-radius: 0.4rem;
+      border-radius: 0.4rem;
     }
 
     ::-webkit-scrollbar-thumb {
-        background-color: var(--scrollbar-color);
+      background-color: var(--scrollbar-color);
     }
 
   </style>
 
   <div class="col main">
-    <h3>Processes</h3>
-    <div class="list">
+    <h2>Using the Widget Api</h2>
+    <h3 class="processes-heading">Displaying the user's processes</h3>
+    <div class="processes-list">
+    </div>
+
+    <h3>Displaying user info</h3>
+    <div class="user-info">
+      <p>State: <span class="state"></span></p>
+      <p>Username: <span class="username"></span></p>
+    </div>
+
+    <h3>Getting the searchbox value</h3>
+    <div class="searchbox-info">
+      The string below will update as you type in the Home tab search box: <br>
+      "<span class="box"></span>"
     </div>
   </div>
 
-  <div class="col">
-    <h3>Second column content</h3>
-    searchbox value:
-    "<span class="box"></span>"
+  <div class="col second">
+    <h2>Second column</h2>
+    <p>This column will disappear as you press the "Collapse" button in the top right of the app.</p>
   </div>
 `;
 
 export class MainComponent extends HTMLDivElement {
   constructor() {
     super();
 
-    this.attachShadow({ mode: 'open' });
+    this.attachShadow({ mode: "open" });
     this.shadowRoot.appendChild(template.content.cloneNode(true));
-    const listRoot = this.shadowRoot.querySelector('.list');
 
-    api.processList$
-    .subscribe(processes => {
-      listRoot.innerHTML = '';
+    const processesListContainer =
+      this.shadowRoot.querySelector(".processes-list");
+
+    combineLatest(
+      api.processList$,
+      (api as any).jobList$ as Observable<any> // TODO: remove any
+    ).subscribe(([processList, jobList]) => {
+      processesListContainer.innerHTML = "";
+      let jobIdentifier = "";
+
+      processList.forEach((process) => {
+        let jobStatus = 1000; // TODO: replace with enum
 
-      processes.forEach(p => {
-        const el = document.createElement('div');
-        el.className = 'process';
-        el.innerHTML = p.name;
-        listRoot.appendChild(el);
+        jobList.forEach((jobDetails) => {
```

Comment:
This can be simplified with [find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find).